projects/core/src/occ/adapters/product/converters/occ-product-search-page-normalizer.ts

```diff
@@ -86,7 +87,9 @@ export class OccProductSearchPageNormalizer
       target.facets = target.facets.map((facetSource: Facet) => {
         const { topValues, ...facetTarget } = facetSource;
         facetTarget.topValueCount =
-          topValues?.length > 0 ? topValues.length : this.DEFAULT_TOP_VALUES;
+          topValues && topValues.length > 0
```

Comment:
Is there a difference for strict mode between

```
topValues?.length > 0
```

and 

```
topValues?.length
```

Reply:
`topValues?.length > 0` has error, because `topValues` can be undefined, which does not has `length`